scripts/cli/build_cli_release.ps1

```diff
@@ -15,6 +15,13 @@ $CARGO_PATH="crates\$CRATE_NAME\Cargo.toml"
 # Get the version of the CLI from its Cargo.toml.
 $VERSION = Get-Content $CARGO_PATH | Select-String -Pattern '^\w*version = "(\d*\.\d*.\d*)"' | % {"$($_.matches.groups[1])"}
 
+# Install the developer tools
+PowerShell -ExecutionPolicy Bypass -File scripts/windows_dev_setup.ps1
+
+# Note: This is required to bypass openssl isssue on Windows.
+echo "VCPKG_ROOT=$env:VCPKG_INSTALLATION_ROOT" | Out-File -FilePath $env:GITHUB_ENV -Append
```

Comment:
FYI: this references a `Github_env` environment variable, which doesn't make sense in this context, right?

Reply:
@JoshLind  Yeah, thats probably not needed. IIUC, that line is just setting up the environment variable? I think I can just set it by `$Env:VCPKG_ROOT = 'C:\vcpkg\'`?